runbotics/runbotics-scheduler/src/database/secret/create-secret.schema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+
+// just an example of zod usage
```

Comment:
maybe leave TODO

is it for describing the requirements specification of the passwords? like special characters, length, etc?